pypreprocess/nipype_preproc_spm_utils.py

```diff
@@ -1714,26 +1877,10 @@ def finalize_report():
 
     # run DARTEL
     preproc_params.update(backup_params)
-    if newsegment:
-        subjects = _do_subjects_newsegment(
+    if dartel:
+        subjects = _do_subjects_dartel(
             subjects, scratch, n_jobs=n_jobs, do_dartel=dartel,
             **preproc_params)
 
-    # apply standard normalization after newsegment ?
-    if normalize and newsegment and not dartel:
-        for stage in ["realign", "coregister", "slice_timing"]:
-            preproc_params[stage] = False
-        preproc_params["hardlink_output"] = True
-        for param in preproc_params.keys():
-            if param not in ["fwhm", "anat_fwhm", "func_write_voxel_sizes",
-                             "anat_write_voxel_sizes", "caching", "report",
-                             "hardlink_output", "smooth_software"]:
-                preproc_params.pop(param)
-        subjects = Parallel(n_jobs=n_jobs)(delayed(_do_subject_normalize)(
-            subject_data, **preproc_params) for subject_data in subjects)
-        # final hard link
-        for subject_data in subjects:
-            subject_data.hardlink_output_files(final=True)
-
     finalize_report()
     return subjects
```

Comment:
Could we have a test that takes an anat image as input and checks that tpms are generated and  / or normalized ?